package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@roadiehq/backstage-plugin-github-pull-requests",
-  "version": "0.7.9",
+  "version": "1.0.0",
```

Comment:
like in the other PR I'm unsure if this is definitely major, for example similar changes in the upstream haven't been marked as such AFAIK ?

Reply:
There are two reasons why I want to bump all of these to 1.0.0:
1. Semver considers `0.x.y` as a special case and all `x` level bump are actually majors, `y` level bumps are minors. There are no patches in the strict sense of the word so bug fixes would always be minors at the least.
2. This is actually a major bump, we rely on `EntityContextProvider` to be present in the wrapping application now. The API is backwards compatible but the on runtime state mgmt isn't.

Reply:
Backstage core team play fast and loose with versioning. I feel like it would be ok if we do to..

Reply:
> Semver considers 0.x.y as a special case and all x level bump are actually majors
Sure, and it's for this reason the upstream has decided to make their corresponding changes "minor" (even though they're major). I would ordinarily tend to agree and move to formal semver but I was inclined to follow suit with respect to the upstream.
> This is actually a major bump, we rely on EntityContextProvider to be present
I suppose I had assumed this to be part of the furniture for a while now but you're right, definitely major (... but we have the option of considering major minor).
TL;DR I withdraw any objections.